### PR TITLE
Write to stdout by default

### PR DESCRIPTION
**Pull request by mgrebenets**
As discussed in https://github.com/fastlane/sigh/issues/268 trying to minimize the level of noise from the `resign.sh` script.

It still outputs warnings when existing signature is replaced, but that comes from `codesign` and probably should just stay as is.

I'll squash commits if it gets approved.

### CODE REVIEW

File: lib/assets/resign.sh [hunk 1]
```diff
@@ -92,65 +107,68 @@ NESTED_APP_REFERENCE_KEYS=(":WKCompanionAppBundleIdentifier" ":NSExtension:NSExt
...
+# Log the options
+for provision in ${RAW_PROVISIONS[@]}; do
+    if [[ "$provision" =~ .+=.+ ]]; then
+        log "Specified provisioning profile: '${provision#*=}' for bundle identifier: '${provision%%=*}'"
+    else
+        log "Specified provisioning profile: '$provision'"
+    fi
+done
+[[ -n "${DISPLAY_NAME}" ]] && log "Specified display name: '$DISPLAY_NAME'"
+[[ -n "${ENTITLEMENTS}" ]] && log "Specified display name: '$ENTITLEMENTS'"
```
Comment (asfalcone) [inline lib/assets/resign.sh]: It looks like lines 150 - 154 include some copy-pasted logging. Should these be as they were above?

**Comment by asfalcone** [comment]
@mgrebenets This looks pretty good to me. Just the 1 comment about copy-pasted logging. Then please squash and I'll merge 🍪

**Comment by mgrebenets** [comment]
Fixed copy paste and squashed all into 1.

**Comment by asfalcone** [comment]
Thanks! 🎉

**Pull request merged**